remoteit/desktop
# [DESK-941] upgrade dependencies

### Description

upgrade dependencies


### Test plan



1. run `npm install`
2. start the application 
3. You should see working fine

### Ticket
https://remoteit.atlassian.net/browse/DESK-941

I tested in windows and mac in development and using a build of that
